web/src/components/PureMarkdownContent.tsx

```diff
@@ -40,7 +40,6 @@ function customRemarkPlugin():
         switch (node.name) {
           case "cannabisLocationAlert":
           case "greenBox":
-          case "infoAlert":
```

Comment:
Why are we removing `infoAlert` here?

Reply:
Great question! So this story is to remove the header properties..  data.hProperties = { header: node.attributes.header };  And since we dont need it anymore and if its a infobox it can be removed.. Did that make sense?